run_linux.go

```diff
@@ -2721,6 +2728,10 @@ func getSecretMount(tokens []string, secrets map[string]define.Secret, mountlabe
 		envFile = tmpFile.Name()
 		ctrFileOnHost = tmpFile.Name()
 	case "file":
+		containerWorkingDir, err := b.store.ContainerDirectory(b.ContainerID)
```

Comment:
Since we already fetch containerWorkingDir at `setupMounts(` I think this could be part of `RunMountSources` if we rename it to `RunMountInfo` or `RunMountAttributes`.

Reply:
I'd prefer to calculate this here, since the `containerWorkingDir` isn't an exclusive attribute to run mounts. I feel like it would be cleaner to only put fields that only deal with run mounts in `RunMountSources`, but if anyone else feels strongly about this I'd be willing to reconsider.